lib/phlex/translation.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Phlex
+	module Translation
+		def self.included(view)
+			view.extend(ClassMethods)
+		end
+
+		module ClassMethods
+			attr_writer :translation_path
+
+			def translation_path
+				@translation_path ||= begin
+					source_location = const_source_location(name).first # Get view absolute file path
+					relative_path = source_location.split(Phlex.configuration.phlex_path).last # Get relative path to the phlex path
+					relative_path.delete_prefix("/").delete_suffix(".rb").tr("/", ".")
+				rescue StandardError
+					# When class doesn't have file location, eg. Class.new(Phlex::View), the const_source_location doesn't work,
+					# it raises exception. This rescue ensures the `translate`  will work in this situation.
+					# `t("hello")` or `t(".hello")` will lookup for "en.hello" translation
+					""
```

Comment:
In the future, we can add a warning here:
```suggestion
          Phlex.configuration.logger.warn "Cannot evaluate view translate path ...`
					""
```